RELEASE/scripts/autoscend/iotms/mr2020.ash

```diff
@@ -214,7 +214,7 @@ void auto_burnPowerfulGloveCharges()
 boolean auto_canFightPiranhaPlant()
 {
 	int numMushroomFights = (in_plumber() ? 5 : 1);
-	if (auto_is_valid($item[packet of mushroom spores]) &&
+	if (is_unrestricted($item[packet of mushroom spores]) &&
 			get_campground() contains $item[packet of mushroom spores] &&
```

Comment:
We can fight the piranha plant in G-Lover since the garden can be planted before we ascend.